plugins/influxdb/plugin.go

```diff
@@ -0,0 +1,22 @@
+package influxdb
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/schema"
+)
+
+func New() schema.Plugin {
+	return schema.Plugin{
+		Name: "influxdb",
+		Platform: schema.PlatformInfo{
+			Name:     "InfluxDB",
+			Homepage: sdk.URL("https://docs.influxdata.com/influxdb/"),
```

Comment:
Is there a reason why we are not just linking to https://www.influxdata.com/?

Reply:
I see no reason. Thanks for suggesting,i have update it